src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -804,15 +806,15 @@ public void execute() throws GitException, InterruptedException {
     public void clean(boolean cleanSubmodule) throws GitException, InterruptedException {
         reset(true);
 	String cmd = "-fdx";
-	if (cleanSubmodule) cmd = "-ffdx";
-
+    if (cleanSubmodule) cmd = "-ffdx";
```

Comment:
Please don't make whitespace changes.  Even as ugly as the formatting is, we retain the formatting for existing code so that pending pull requests are not blocked by conflicts due to whitespace changes.

Reply:
Sorry, fixed now.